lib/capybara.rb

```diff
@@ -48,7 +48,7 @@ class << self
     # [ignore_hidden_elements = Boolean]  Whether to ignore hidden elements on the page (Default: true)  
     # [automatic_reload = Boolean]        Whether to automatically reload elements as Capybara is waiting (Default: true)  
     # [save_and_open_page_path = String]  Where to put pages saved through save_and_open_page (Default: Dir.pwd)  
-    #
+    # [first_default_waiting = Boolean]   Whether Node#first defaults to Capybara waiting behavior for at least 1 element to match (Default: false) 
```

Comment:
`make_first_to_wait_by_default` isn't good, however, as waiting will not happen if wait is set to 0.